refactor: Apply Wikipedia's style of bolding to multiple pages (#2662)
privacyguides/privacyguides.org: docs/encryption.md

description: Encryption of data is the only way to control who can access it. These tools allow you to encrypt your emails and any other files.
cover: encryption.webp
---

> **Encryption of data** is the only way to control who can access it. If you are currently not using encryption software for your hard disk, emails or files, you should pick an option here.

Reviewer: ```suggestion
**Encryption** is the only secure way to control who can access your data. If you are currently not using encryption software for your hard disk, emails, or files, you should pick an option here.
```
